src/App.js

```diff
@@ -1,8 +1,13 @@
-import React, { Component } from "react";
-import styles from "./App.css";
-import Header from "./components/Header/Header";
-import Main from "./components/Main/Main";
-import Footer from "./components/Footer/Footer";
+import React, { Component } from 'react';
+import { Provider } from 'react-redux';
+
+import styles from './App.css';
+import Header from './components/Header/Header';
+import Main from './components/Main/Main';
+import Footer from './components/Footer/Footer';
+import store from './store/initialState';
+
+// console.log('store', store);
 
 class App extends Component {
```

Comment:
You can also change App to a functional component.

Reply:
done